src/routes/backoffice.ts

```diff
@@ -376,6 +376,8 @@ export const processQueuedBooking = async (
     booking.amount.value = -Math.abs(booking.amount.value);
   }
 
+  booking.account_id = person.accounts[0].id;
```

Comment:
I think I asked this in a different pull request as well .. 
but if the booking does not have an account_id in their specs .. why are we adding it and does it not break the return values further down the line ?